advanced_functionality/multi-container-endpoint/direct-invocation/multi-container-direct-invocation.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<p>Missing underscore - <code>instance_count</code>  instead of <code>instance count</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>